src/io/openshift/Plugins.groovy

```diff
@@ -1,8 +1,14 @@
 package io.openshift
-import io.openshift.plugins.*
+
+import io.openshift.plugins.analytics
+import io.openshift.Globals
 
 class Plugins implements Serializable {
     static def register() {
+      Globals.plugins["analytics"] = Globals.plugins["analytics"] ?: [disabled: false]
```

Comment:
// please add a comment that analytics is enabled by default

Reply:
ok, will do, also will add this new api in readme